tests/acceptance/30_generic_methods/network/http_request_content_headers.cf

```diff
@@ -22,8 +22,8 @@ body common control
 bundle agent init
 {
   vars:
-    "existant_url"          string => "http://www.ncf.io";
-    "nonexistant_url"       string => "http://www.ncf.io/testurl";
+    "existant_url"          string => "http://www.rudder.io";
```

Comment:
could you use repository.rudder.io instead?

Reply:
sure